cupy/cuda/compiler.py

```diff
@@ -2,35 +2,31 @@
 import os
 import re
 import shutil
-import subprocess
-import sys
 import tempfile
 
 import six
 
 from cupy.cuda import device
 from cupy.cuda import function
+from cupy.cuda import nvrtc
 
+_nvrtc_version = None
 
-_nvcc_version = None
 
+def _get_nvrtc_version():
+    global _nvrtc_version
+    if _nvrtc_version is None:
+        _nvrtc_version = nvrtc.getVersion()
 
-def _get_nvcc_version():
-    global _nvcc_version
-    if _nvcc_version is None:
-        cmd = ['nvcc', '--version']
-        _nvcc_version = _run_nvcc(cmd, '.')
-
-    return _nvcc_version
+    return _nvrtc_version
 
 
 def _get_arch():
     cc = device.Device().compute_capability
-    return 'sm_%s' % cc
+    return 'compute_%s' % cc
```

Comment:
Sorry for inconvenience, I replied to it in my comment at this PR:
https://github.com/cupy/cupy/pull/33#discussion_r113092304

> compute_xx is used to specify a virtual GPU architecture, while sm_xx is used to specify the actual real architecture for which a given code/PTX will be compiled. NVRTC generates PTX file for virtual architecture, and then the CUDA driver converts the PTX file for each real GPU architecture by generating an optimized binary.

So in this case, we need to use compute_xx for the option to NVRTC.